docs/Features/Vanity.md

```diff
@@ -0,0 +1,34 @@
+# Vanity
+Satisfy all of your cosplay needs with easy-to-use, non-intrusive vanity system.
```

Comment:
Should be "**an** easy to use". Might be worth mentioning that unlike other implementations, this has no gameplay sideeffects (ex. with GB armor sets)